public/pages/Administration/components/webhook/WebhookForm.tsx

```diff
@@ -0,0 +1,286 @@
+          />
+          <TextArea
+            className="c-webhook-form__content"
+            field="content"
+            label="Content"
+            afterLabel={<HoverInfo text="You can use Go template formatting with many properties here" onClick={this.showModal} />}
+            value={this.state.content}
+            onChange={this.setContent}
+            placeholder="Request body"
+          />
+          <Input field="http_method" label="HTTP Method" value={this.state.http_method} onChange={this.setHttpMethod} placeholder="POST" />
+          <Field label="Additional HTTP Headers" afterLabel={<HoverInfo text="Those headers are sent in the request when the webhook is triggered" />}>
+            <VStack spacing={2}>
+              {Object.entries(this.state.additional_http_headers).map(([header, value]) => (
+                <HttpHeader key={header} header={header} value={value} onEdit={this.setHttpHeader} onRemove={this.removeHttpHeader} />
+              ))}
+              <HttpHeader onEdit={this.setHttpHeader} allHeaders={allHeaders} />
+            </VStack>
+          </Field>
+          {(this.state.url || this.state.content) && (
+            <Field label="Preview" className="c-webhook-form__preview">
+              {this.state.preview === null ? (
+                <p className="text-muted">Failed to load preview</p>
+              ) : this.state.preview === undefined ? (
+                <Loader className="text-center" text="Loading preview" />
+              ) : (
+                <VStack className="bg-gray-50 rounded-md p-2" spacing={2}>
+                  {this.state.url && (
+                    <div>
+                      <h3 className="text-bold mb-1">URL</h3>
+                      <pre>{this.state.preview.url.value ? this.state.preview.url.value : this.state.preview.url.error}</pre>
+                      {this.state.preview.url.message && <p className="text-muted">{this.state.preview.url.message}</p>}
+                    </div>
+                  )}
+                  {this.state.content && (
+                    <div>
+                      <h3 className="text-bold mb-1">Content</h3>
+                      <pre>{this.state.preview.content.value ? this.state.preview.content.value : this.state.preview.content.error}</pre>
+                      {this.state.preview.content.message && <p className="text-muted">{this.state.preview.content.message}</p>}
+                    </div>
+                  )}
+                </VStack>
+              )}
+            </Field>
+          )}
+          <HStack>
+            <Button variant="primary" onClick={this.handleSave}>
+              Save
+            </Button>
+            <Button onClick={this.handleCancel} variant="tertiary">
+              Cancel
+            </Button>
+          </HStack>
+        </Form>
+        <WebhookTemplateInfo type={this.state.type} isModalOpen={this.state.isModalOpen} onModalClose={this.hideModal} />
```

Comment:
rename component to WebhookTemplateInfoModal to follow the convention

Reply:
It will be done